wangle/acceptor/TransportInfo.cpp

```diff
@@ -86,6 +86,10 @@ int64_t TransportInfo::readRTT(const folly::AsyncSocket* sock) {
 #endif
 }
 
+#ifdef __APPLE__
+#define TCP_INFO TCP_CONNECTION_INFO;
```

Comment:
You have a typo on this line: please remove the semicolon in the define.

Reply:
Indeed! How embarassing...